### ISSUE TITLE
How to identify both cats and dogs

### ISSUE THREAD

**Issue by seraphD** [keras-team/keras-applications#126]
Just like title. Suppose I have a pic that contain both cats and dogs and I want to recognize them all. Then should I set up another class or some other ways?

**Comment by 2sang** [comment]
You could either try the multi-label approach for that or add a new class like 'cat-and-dog' class.
And you may want to ask this at Stackoverflow for a more elaborated answer. :)

**Comment by taehoonlee** [comment]
@seraphD, Please refer to the [official docs](https://keras.io/applications/#usage-examples-for-image-classification-models).
